blueocean-dashboard/src/main/less/core.less

```diff
@@ -390,9 +390,11 @@ table.highlight {
     width: 100%;
     height: 100%;
     display: block;
-    padding: 0.5em 0;
+    padding: 0.5em 0.8em 0.5em 0;
     text-decoration: none;
     color: inherit;
+    line-height: 22px; // force height to fill space to ensure clicks are "captured"
+    .text-overflow;
```

Comment:
Is that something you wanted to incorporate into a future change to the table? In jenkinsci/design-language#118 the `text-overflow` mixin is still being applied to all `td` and `td` elements nested in a `.jdl-table`